docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  fxa-oauth-console:
+    build:
+      context: .
+      dockerfile: Dockerfile-build
+    command: node bin/server.js
+    image: mozilla/fxa-oauth-console
+    ports:
+      - "10137:10137"
```

Comment:
See if you can start up the pm2 servers locally, run oauth with MySQL via these instructions: https://github.com/mozilla/fxa-local-dev#oauth-console-developer-accounts 

and try to login to the console using an `@mozilla.com` email